Align touchcancel and pointercancel examples (#30698)
mdn/content: files/en-us/web/api/element/touchcancel_event/index.md

{{APIRef}}

The `touchcancel` event is fired when one or more touch points have been disrupted in an implementation-specific manner.

Some examples of situations that will trigger a `touchcancel` event:

- A hardware event occurs that cancels the touch activities. This may include, for example, the user switching applications using an application switcher interface or the "home" button on a mobile device.
- The device's screen orientation is changed while the touch is active.
- The browser decides that the user started touch input accidentally. This can happen if, for example, the hardware supports palm rejection to prevent a hand resting on the display while using a stylus from accidentally triggering events.
- The {{cssxref("touch-action")}} CSS property prevents the input from continuing.

> - When the user interacted with too many simultaneous fingers the browser can fire this event for all existing pointers (even if they are still touching the screen).

Reviewer: ```suggestion
- When the user interacts with too many fingers simultaneously, the browser can fire this event for all existing pointers (even if the user is still touching the screen).
```
